api/eds.proto

```diff
@@ -205,9 +192,9 @@ message ClusterLoadAssignment {
   Policy policy = 4;
 }
 
-message LoadAssignmentResponse {
-  bytes version_info = 1;
-  repeated ClusterLoadAssignment clusters = 2;
```

Comment:
They come from a separate stream initiated from EDS to the management server, either via EDS or encapsulated in ADS. In general, the API doesn't guarantee same management server affinity for load reporting as load assignment in this proposal (even ADS is only for CDS/LDS/RDS/basic EDS assignment, so won't have this either).

Do you foresee any issues here?